docs/instruments/bmi.json

```diff
@@ -0,0 +1,532 @@
+{
+  "schema": {
+    "meta": {
+      "createdBy": "Zaliqa Rosli",
+      "createdOn": "2019-02-19",
+      "createdWith": "",
+      "defaultLanguage": "en-CA",
+      "derivedFrom": "",
+      "importedFrom": "bmi.linst",
+      "includeMetaDataFields": true,
+      "instrumentVersion": "",
+      "lastUpdateOn": "",
+      "longName": {
+          "en-CA": "BMI Calculator"
+      },
+      "multilingual": false,
+      "previousVersion": "",
+      "schemaVersion": "v0.0.1-dev",
+      "shortName": "bmi"
+    },
+    "fields": {
+      "unit_classification" : {
+        "type": "enum",
+        "description": {
+            "en-CA": "Choose unit of measurement"
+        },
+        "options": {
+          "values": [
+            {
+              "description": {
+                "en-CA": "Metric"
+              },
+              "value": "metriC"
+            },
+            {
+              "description": {
+                "en-CA": "Imperial"
+              },
+              "value": "imperial"
+            }
+          ],
+          "allowMultipleValues": false,
+          "requireResponse": true,
+          "readonly": false,
+          "hideInSurvey": false
+        },
+        "rules": {
+          "requireIf": "",
+          "displayIf": "",
+          "disableIf": "" 
+        }
+      },
+      "height_feet" : {
+        "type": "decimal",
+        "description": {
+            "en-CA": ""
+        },
+        "options": {
+          "minValue": "",
+          "maxValue": "",
+          "requireResponse": true,
+          "readonly": false,
+          "hideInSurvey": false,
+          "showDesc": false
+
+        },
+        "rules": {
```

Comment:
Rules for these 5 fields  (and also their `_status` fields?) : 
* height_feet
* height_inches
* weight_lbs
* height_cms
* weight_kgs

are found in the *.rules file for this Linst : 
https://github.com/aces/Loris/blob/minor/docs/instruments/bmi.rules

Reply:
thanks, this implementation is still not complete so that's why there are things missing

Reply:
@christinerogers please review the scores and rules added in the last commit